### PR TITLE
Improve docstrings of Animations

### PR DESCRIPTION
**Pull request by timhoffm**
## PR Summary

Giving some more information and adapting to current docstring conventions.

### CODE REVIEW

**Comment by dopplershift** [comment]
Waiting on CI.

File: lib/matplotlib/animation.py [hunk 1]
```diff
@@ -1293,14 +1293,30 @@ def _end_redraw(self, evt):
                                                        self._handle_resize)
 
     def to_html5_video(self, embed_limit=None):
-        '''Returns animation as an HTML5 video tag.
+        """
+        Convert the animation to a HTML5 ``<video>`` tag.
```
Comment (QuLogic) [inline lib/matplotlib/animation.py]: This is still 'an', not 'a'. There is no reading of HTML5 that doesn't start with a vowel sound.

File: lib/matplotlib/animation.py [hunk 2]
```diff
@@ -1293,14 +1293,30 @@ def _end_redraw(self, evt):
...
+        ----------
+        embed_limit : float, optional
+            Limit, in MB, of the returned animation. No animation is created
+            if the limit is exceeded.
+            Defaults to :rc:`animation.embed_limit` = 20.0.
+
+        Returns
+        -------
+        video_tag : str
+            A HTML5 video tag with the animation embedded as base64 encoded
```
Comment (QuLogic) [inline lib/matplotlib/animation.py]: Also, 'An' here.

**Comment by timhoffm** [comment]
Comments handled.

**Pull request merged**